scripts/cmake/Modules/LibAddPlugin.cmake

```diff
@@ -637,6 +637,8 @@ function (add_plugin PLUGIN_SHORT_NAME)
 
 	if (BUILD_SHARED)
 		add_library (${PLUGIN_NAME} MODULE ${ARG_SOURCES} ${ARG_OBJECT_SOURCES})
+		set_target_properties (${PLUGIN_NAME} PROPERTIES PREFIX "libelektra")
```

Comment:
```suggestion
```

I'm not sure that's correct. AFAIK `PREFIX` normally is `lib` on Linux systems (because shared libraries are called `libSOME-NAME.so`) and an empty string on Windows (since they use `SOME-NAME.dll`). Not sure exactly what macOS does...